homework/sql_practice/homework/database.py

```diff
@@ -0,0 +1,122 @@
+from datetime import datetime
+from databaseconnection import DatabaseConnection
+from setup import set_table
+
+from users_dict import users_dict, update_user_dict, bad_user_dict
+
+
+class DataBaseParser:
+    DT = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+
+    @staticmethod
+    def _cleaned_up():
+        with DatabaseConnection() as cursor:
+            cursor.execute('DROP TABLE cart_details')
+            cursor.execute('DROP TABLE cart')
+            cursor.execute('DROP TABLE users')
+
+    @staticmethod
+    def _setup():
+        with DatabaseConnection() as cursor:
+            cursor.execute(set_table)
+
+    @staticmethod
+    def create_user(user_info) -> None:
+        """Creation user in db 'users' with values 'name', 'email', 'registration time' """
+        with DatabaseConnection() as cursor:
+            cursor.execute(
+                """INSERT INTO users (name, email, registration_time)
+                 VALUES (%(name)s, %(email)s, %(registration_time)s)""",
+                user_info, )
+
+    @staticmethod
+    def read_user_info(user_id) -> str or None:
```

Comment:
невпевнений що так можна писати 
```suggestion
    def read_user_info(user_id) -> str:
```